lib/usd/ui/layerEditor/qtUtils.cpp

```diff
@@ -35,6 +35,14 @@ QPixmap QtUtils::createPNGResPixmap(QString const& in_pixmapName, int width, int
     if (pixmapName.left(2) != resourcePrefix) {
         pixmapName = resourcePrefix + pixmapName;
     }
+
+    // Note: createPNGResPixmap calls QtUtils::createPNGResPixmap, but that is
+    //       *not* the function below but rather MayaQtUtils::createPixmap, since
+    //       these functions are virtual.
+    //
+    //       The MayaQtUtils version calls MQtUtil::createPixmap which calls
+    //       QmayaQtHelper::createPixmap which generates the scaled image name
+    //       by adding the _150 or _200 suffix as necessary.
     return createPixmap(pixmapName, width, height);
```

Comment:
Added the explanation how the scaled image names are generated.

Reply:
Thank you.